setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='0.0.8',
+    version='0.1.0',
```

Comment:
Is there a plan to get this to version 1? It seems like it has proved itself in production now.